init/services/HestiaKERNEL/Is_Unicode.sh

```diff
@@ -46,7 +46,7 @@ HestiaKERNEL_Is_Unicode() {
 
         # execute
         case "$1" in
-        *[!0123456789,\ ]*)
+        *[!0123456789,\ \-\+]*)
```

Comment:
MacOS and BSD OS type uses an older version of terminal so it needs a octet byte parsing. For now, this is placed temporarily for invalid characters.